Dockerfile

```diff
@@ -12,9 +12,14 @@ RUN mkdir -p ${TMP_DIR} && \
     # already create volume dir to later have correct rights
     mkdir ${TMP_DIR}/data
 
+RUN groupadd -g 1000 zeebe && \
+    adduser -u 1000 zeebe --system --ingroup zeebe
+
 ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini ${TMP_DIR}/bin/tini
 COPY docker/utils/startup.sh ${TMP_DIR}/bin/startup.sh
 RUN chmod +x -R ${TMP_DIR}/bin/
+RUN chgrp 0 ${TMP_DIR}/bin/startup.sh && chmod 0775 ${TMP_DIR}/bin/startup.sh
```

Comment:
I essentially moved this line, so I didn't think too much about it and trusted whoever added it knew why - but you're right, 0 is root so no idea why we're changing the group to root :thinking: We're anyway later changing it again to 1000, so yeah, not sure. I guess I should test this stuff on OpenShift since this is where the requirement came from